sos/report/__init__.py

```diff
@@ -1257,7 +1257,7 @@ def setup(self):
                 plug.manifest.add_field('setup_end', end)
                 plug.manifest.add_field('setup_time', end - start)
             except KeyboardInterrupt:
-                raise KeyboardInterrupt
+                raise KeyboardInterrupt  # pylint: disable=raise-missing-from
```

Comment:
Just a `raise` would be enough but that triggers `try-except-raise`.

Removing 
```
except KeyboardInterrupt:
                raise KeyboardInterrupt
```
altogether would mean `KeyboardInterrupt` would fall into 
```
except Exception:
                self.handle_exception(plugname, "setup")
```

which handles the exception differently.